airflow/datasets/metadata.py

```diff
@@ -33,7 +33,18 @@ class Metadata:
 
     uri: str
     extra: dict[str, Any]
-
-    def __init__(self, target: str | Dataset, extra: dict[str, Any]) -> None:
+    alias: list[DatasetAlias | str] | None
```

Comment:
This should probably be called `aliases` (plural) instead. Is `None` special here, or we could just use an empty list instead?

Reply:
yep, was trying to align with the example, but I think we should use `aliases` as well. just updated it. thanks!